src/content/reference/cli.mdx

```diff
@@ -572,6 +572,10 @@ You need to be [logged in](reference/cli.mdx#context) to Okteto before running t
 Only users with administration privileges can destroy a global preview environment.
 If the manifest includes `destroy` commands, they will be executed as part of this command.
 
+| Options                                   |    Type    | Description                                                                                                                                                                                                                                               |
+| :---------------------------------------- | :--------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _--wait_                                  |   (bool)   | Wait until the preview deletion finishes (defaults to false)                                                                                                                                                                                            |
```

Comment:
```suggestion
| _--wait_                                  |   (bool)   | Wait until the preview deletion finishes (defaults to true)                                                                                                                                                                                            |
```

The default value is set to true to avoid breaking changes.